### PR TITLE
Add trozet to owners

### PR DESCRIPTION
**Pull request by trozet**
Signed-off-by: Tim Rozet <trozet@redhat.com>

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -7,7 +7,7 @@ reviewers:
   - dcbw
   - JacobTanenbaum
   - juanluisvaladas
-  - pecameron
+  - trozet
```
Comment (danwinship) [inline OWNERS:10]: This breaks the sorting...

**Review by danwinship — COMMENTED** [review]
You should update openshift/sdn too for consistency. Also, the rest of Feng's team was supposed to get added as `reviewers` to these, but then no one did that...

File: OWNERS (hunk 1 above)
Comment (danwinship) [reply OWNERS:10]: also maybe add @fepan as an approver but not a reviewer (for emergency merges when no one else is around?)

Comment (trozet) [reply OWNERS:10]: I didn't know it was sorted. Will do.

**Comment by danwinship** [comment]
/hold
(to stop it from retesting again)

**Pull request merged**